src/databricks/labs/ucx/cli.py

```diff
@@ -179,17 +179,35 @@ def validate_external_locations(w: WorkspaceClient, prompts: Prompts):
         webbrowser.open(f"{w.config.host}/#workspace{path}")
 
 
-@ucx.command
-def ensure_assessment_run(w: WorkspaceClient, *, run_as_collection: bool = False, a: AccountClient | None = None):
-    """ensure the assessment job was run on a workspace"""
-    workspace_contexts = _get_workspace_contexts(w, a, run_as_collection)
-    # if running the cmd as a collection, don't wait for each assessment job to finish as that will take long time
-    skip_job_status = bool(run_as_collection)
+def _workflow_trigger(
```

Comment:
Could you revert and create some duplicate code for both commands? 

Reason:
1. We might refactor the `get_workspace_contexts` bits #2507 
2. We want to keep logic in the `cli.py` to a minimum. If there is too much (duplicate) logic, then we can move it into a dedicated module.

Reply:
Updated.